conandata.yml

```diff
@@ -0,0 +1,16 @@
+fetch_sources: 'git/master'
+sources:
+  git:
+    master:
+      url: "https://github.com/riebl/artery"
+      args:
+        - "--recurse-submodules"
+        - "--branch"
+        - "master"
+        - "--depth"
+        - "1"
+requirements:
+  boost: "1.86.0"
+  cryptopp: "8.9.0"
+  geographiclib: "2.3"
+  zeromq: "4.3.5"
```

Comment:
zeromq is only required for the communication with OpenTrafficSim (OTS). Can we declare it as an optional dependency?